lib/api/1.1/northbound/nodes.js

```diff
@@ -569,9 +577,12 @@ function nodesRouterFactory (
     router.delete('/nodes/:identifier/workflows/active', rest(function (req) {
         return waterline.nodes.needByIdentifier(req.params.identifier)
         .then(function (node) {
-            return taskGraphProtocol.cancelTaskGraph({ target: node.id });
+            return workflowApiService.findActiveGraphForTarget(node.id);
+        })
+        .then(function(graph) {
+            return workflowApiService.cancelTaskGraph(graph.instanceId);
```

Comment:
if there is no active workflow found, does this `500` error or pass through with a successful cancel? (i.e. what does `workflowApiService.cancelTaskGraph` do when graph is empty? Will there be a graph.instanceId on that object no matter what, or is that a bad assumption here?